cmd/habitatctl/commands/start.go

```diff
@@ -24,13 +24,13 @@ func parseFlags(args []string) ([]string, []string) {
 
 var commName string
 
-var ipfsCmd = &cobra.Command{
+/*var ipfsCmd = &cobra.Command{
```

Comment:
wait why are we commenting this out? also delete it?

Reply:
It was unused and I was unsure if we wanted to keep it around? Do you think we should still have this command?

Reply:
hmm, i think we can leave it commented out for now. at some point might come in handy but not planning to maintain it rn